Exiled.Events/Patches/Events/Player/TogglingRadio.cs

```diff
@@ -0,0 +1,85 @@
+// -----------------------------------------------------------------------
+// <copyright file="TogglingRadio.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+namespace Exiled.Events.Patches.Events.Player
+{
+
+    using Exiled.API.Features;
+    using Exiled.Events.Attributes;
+    using Exiled.Events.EventArgs.Player;
+#pragma warning disable SA1313 // Parameter names should begin with lower-case letter
+    using HarmonyLib;
+    using InventorySystem.Items;
+    using InventorySystem.Items.Radio;
+    using PluginAPI.Events;
+
+    /// <summary>
+    ///     Patches <see cref="RadioItem.ServerProcessCmd" />.
+    ///     Adds the <see cref="Handlers.Player.TogglingRadio" /> event.
+    /// </summary>
+    [EventPatch(typeof(Handlers.Player), nameof(Handlers.Player.TogglingRadio))]
+    [HarmonyPatch(typeof(RadioItem), nameof(RadioItem.ServerProcessCmd))]
+    internal static class TogglingRadio
+    {
+        private static bool Prefix(RadioItem __instance, RadioMessages.RadioCommand command)
```

Comment:
i will see if @BoltonDev  can make you the Transpiler also yeah ExiledEvent need to be call before NWEvent if it's deniable